compute/admin_guide/waas/deploy_waas/fragments/add-app-policy.adoc

```diff
@@ -9,23 +9,24 @@ NOTE: The combination of *Rule name* and *App ID* must be unique across In-Line
 If you have a Swagger or OpenAPI file, click *Import*, and select the file to load.
 +
 If you do not have a Swagger or OpenAPI file, manually define each endpoint by specifying the host, port, and path.
-+
-image::cwp-42473-add-app-policy.png[scale=10]
 
 .. In *Endpoint Setup*, click *Add Endpoint*.
-
-.. Specify endpoint in your web application that should be protected. Each defined application can have multiple protected endpoints.
++
+Specify endpoint in your web application that should be protected. Each defined application can have multiple protected endpoints.
 
 .. Enter *HTTP host* (optional, wildcards supported).
 +
 HTTP hostnames are specified in the form of [hostname]:[external port].
 +
 The external port is defined as the TCP port on the host, listening for inbound HTTP traffic. If the value of the external port is "80" for non-TLS endpoints or "443" for TLS endpoints it can be omitted. Examples: "*.example.site", "docs.example.site", "www.example.site:8080", etc.
 
-.. Enter *App ports* as the internal port your app listens on (optional, if you selected *Automatically detect ports* while creating the rule). 
+.. Enter *App ports* as the internal port your app listens on.
+ifndef::waas-app-embedded[]
```

Comment:
This condition excludes the below line from appearing in the App-Embedded WAAS page.